src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -2948,6 +3097,15 @@ module Mutations = struct
               ~fee_token ~fee_payer_pk:from ~valid_until ~body ~signature
             |> Deferred.Result.map ~f:Types.UserCommand.mk_user_command)
 
+  let send_snapp =
+    io_field "sendSnappTransaction" ~doc:"Send a Snapp transaction"
```

Comment:
Please change to `sendSnapp` for consistency

Reply:
Done.